cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -1221,12 +1230,68 @@ abstract class ElectrumWalletBase
   void setExceptionHandler(void Function(FlutterErrorDetails) onError) => _onError = onError;
 
   @override
-  String signMessage(String message, {String? address = null}) {
+  Future<String> signMessage(String message, {String? address = null}) async {
     final index = address != null
         ? walletAddresses.allAddresses.firstWhere((element) => element.address == address).index
         : null;
     final HD = index == null ? hd : hd.derive(index);
-    return base64Encode(HD.signMessage(message));
+    final priv = ECPrivate.fromHex(HD.privKey!);
+    String messagePrefix = '\x18Bitcoin Signed Message:\n';
+    return priv.signMessage(utf8.encode(message), messagePrefix: messagePrefix);
+  }
```

Comment:
Please use the previous implementation HD.signMessage since we made changes to the lib to include the rec-id (v) in the signature. Without v the Sign-In on our exchange helper and DFX fails. It appears that the Lib you are using now does not include the rec-id in the sig